tests/Restore-DbaDatabase.Tests.ps1

```diff
@@ -143,6 +143,16 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
         }
     }
 
+    Context "Replace databasename in Restored File, but don't change backup history #5036" {
```

Comment:
Tests are good 👍 

 Should stop your hard work getting broken further on down the line